From abccaf9555f6a49f9a3ddf0db81e813649a0ec09 Mon Sep 17 00:00:00 2001
From: Aleksa Sarai <asarai@suse.de>
Date: Thu, 20 Jul 2017 03:33:37 +1000
Subject: [PATCH] generate: fix build for v1.0.0

There was a very late post-v1.0.0-rc6 hotfix that broke the build
between the two versions. It's not a semantic change, a field was moved
to a sub-struct.

Signed-off-by: Aleksa Sarai <asarai@suse.de>
---
 generate/generate.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/generate/generate.go b/generate/generate.go
index 0eeda5c776bd..848a970824dd 100644
--- a/generate/generate.go
+++ b/generate/generate.go
@@ -486,8 +486,8 @@ func (g *Generator) SetLinuxMountLabel(label string) {
 
 // SetLinuxResourcesDisableOOMKiller sets g.spec.Linux.Resources.DisableOOMKiller.
 func (g *Generator) SetLinuxResourcesDisableOOMKiller(disable bool) {
-	g.initSpecLinuxResources()
-	g.spec.Linux.Resources.DisableOOMKiller = &disable
+	g.initSpecLinuxResourcesMemory()
+	g.spec.Linux.Resources.Memory.DisableOOMKiller = &disable
 }
 
 // SetProcessOOMScoreAdj sets g.spec.Process.OOMScoreAdj.
-- 
2.13.2

